docker/openproblems-python-method-scvis/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM python:3.6
```

Comment:
Should we create an `openproblems-py36` image as a base for this? Seems broader than just scvis.

Reply:
That seems reasonable to me I suppose. Is it just renaming things or would there need to be bigger updates?

Reply:
The question would be whether we follow the structure in the py38 image in which we have a python base container and then an `extras` container built on top that adds stuff like scvis.

Reply:
For 3.6 I reckon it's probably sth that will not be heavily used anymore so a separate image should be enough, no? How much python 3.6 dependencies are flying about?

Reply:
That seems reasonable. Let's just call it openproblems-python36 and leave it at that for now

Reply:
Ok, I renamed this to `openproblems-python36`